isobus/src/isobus_virtual_terminal_server.cpp

```diff
@@ -2601,6 +2615,26 @@ namespace isobus
 		                                                      get_priority());
 	}
 
+	bool VirtualTerminalServer::send_capture_screen_response(uint8_t item, uint8_t path, uint8_t errorCode, uint16_t imageId, std::shared_ptr<ControlFunction> requestor)
```

Comment:
```suggestion
	bool VirtualTerminalServer::send_capture_screen_response(std::uint8_t item, std::uint8_t path, std::uint8_t errorCode, std::uint16_t imageId, std::shared_ptr<ControlFunction> requestor)
```

Reply:
Awg, we need to setup something to prevent this in future. Or I should take more care to it...